bundle/src/main/java/com/jfrog/ide/eclipse/ui/licenses/LicensesTree.java

```diff
@@ -0,0 +1,76 @@
+package com.jfrog.ide.eclipse.ui.licenses;
+
+import java.util.Map.Entry;
+
+import org.eclipse.jface.viewers.ColumnLabelProvider;
+import org.eclipse.swt.widgets.Composite;
+import org.jfrog.build.extractor.scan.DependenciesTree;
+
+import com.jfrog.ide.common.filter.FilterManager;
+import com.jfrog.ide.eclipse.ui.ProjectsMap;
+import com.jfrog.ide.eclipse.ui.SearchableTree;
+
+/**
+ * @author yahavi
+ */
+public class LicensesTree extends SearchableTree {
+
+	private static LicensesTree instance;
+	private DependenciesTree root = new DependenciesTree();;
+
+	public static void createLicensesTree(Composite parent) {
```

Comment:
In ComponentLicenseDetails class the static methods are create and get. In LicensesTree it createLicensesTree or getLicensesTree. I think we should have one convention. I prefer the one in the LicensesTree so I think the ComponentLicenseDetails should be changed. For example in the LicensesFilterDialog also the convention as in the LicensesTree --> createDialogArea

Reply:
I'm changing:
`get` --> `getInstance` To be consistent with the [singletone pattern](getInstance).
`create` --> `createXYZ` to be more clear.